### PR TITLE
Generic and crew-available status readout module

### PR DESCRIPTION
**Pull request by Stalkeros2**
## About The Pull Request
Ninja's health-and-nutrition-and-such status readout module is no longer ninja-exclusive; added to the advanced medical research techweb. As such, its description has been slightly edited to affect its status.
image
## Why It's Good For The Game
This module always felt like it shouldn't be locked behind and thus only use by a relatively rare antagonist: it was, just, super convenient, and useful for uuuh... I'd say anyone on station, to view their accurate health status on the spot. And with the health analyzer module doing the exact same thing, and more, -being able to check other people's health whoa!-, I don't think it'd be out of the norm to make the ninja's module station-available too.
## Changelog
:cl: Stalkeros
add: Status readout module: now generic, additionally station-available.
/:cl:

### CODE REVIEW

File: code/modules/research/techweb/all_nodes.dm [hunk 1]
```diff
@@ -1692,6 +1692,7 @@
 		"mod_defib",
 		"mod_threadripper",
 		"mod_surgicalprocessor",
+		"mod_statusreadout"
```
Comment (Fikou) [inline code/modules/research/techweb/all_nodes.dm:1695]:
```suggestion
		"mod_statusreadout",
```

File: code/modules/research/designs/mechfabricator_designs.dm [hunk 2]
```diff
@@ -1897,6 +1897,16 @@
...
+/datum/design/module/statusreadout
+	name = "Status Readout Module"
+	id = "mod_statusreadout"
+	materials = list(/datum/material/iron = HALF_SHEET_MATERIAL_AMOUNT * 3, /datum/material/glass = HALF_SHEET_MATERIAL_AMOUNT, /datum/material/titanium = SMALL_MATERIAL_AMOUNT * 2)
+	build_path = /obj/item/mod/module/status_readout
+	category = list(
+		RND_CATEGORY_MODSUIT_MODULES + RND_SUBCATEGORY_MODSUIT_MODULES_MEDICAL
+	)
+
+
```
Comment (Fikou) [inline code/modules/research/designs/mechfabricator_designs.dm:1909]:
```suggestion
```

**Review by Fikou — CHANGES_REQUESTED** [review]
This shouldn't render the operation time and number (a ninja subtype should do that), and it shouldn't be in the ninja file
Also add a flatline sound effect on death coward

**Comment by JohnFulpWillard** [comment]
This PR looks done but it's marked as draft, was this ready to review or not? @Stalkeros2

**Comment by Stalkeros2** [comment]
> This PR looks done but it's marked as draft, was this ready to review or not? @Stalkeros2

IRL stuff happened; afaik one of the maintainers (maybe it was you even. Or Fikou. Likely Fikou.) asked me to make the Operation Number and Operation Time parts of the UI ninja-only, so I was trying to do exactly that; and eventually, forgot about this PR. Otherwise, it's entirely done.

**Pull request closed without merging**